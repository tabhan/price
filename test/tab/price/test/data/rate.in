['ITEM000001-2','ITEM000001','ITEM000003-5','ITEM000005','ITEM000005']