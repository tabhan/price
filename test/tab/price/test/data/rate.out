***<没钱赚商店>购物清单***
名称：可口可乐，数量：3瓶，单价：3.00(元)，小计：9.00(元)
名称：羽毛球，数量：5个，单价：1.00(元)，小计：5.00(元)
名称：苹果，数量：2斤，单价：5.50(元)，小计：10.45(元)，节省0.55(元)
----------------------
总计：24.45(元)
节省：0.55(元)
**********************